libtapi is by default limited to be linked to binaries called ld only. When
trying to link it into e.g. our ld64 we get:

ld: cannot link directly with ${EPREFIX}/usr/lib/libtapi.dylib for architecture
x86_64

Drop that limitation in order to be able to use the library.

--- libtapi-1.30/tools/libtapi/CMakeLists.txt.orig	2017-02-12 17:35:55.000000000 +0100
+++ libtapi-1.30/tools/libtapi/CMakeLists.txt	2017-02-12 17:36:17.000000000 +0100
@@ -18,5 +18,5 @@
 
 set_property(TARGET libtapi APPEND_STRING
   PROPERTY
-  LINK_FLAGS " -current_version ${TAPI_VERSION_MAJOR}.${TAPI_VERSION_MINOR}.${TAPI_VERSION_PATCH} -compatibility_version 1 -allowable_client ld"
+  LINK_FLAGS " -current_version ${TAPI_VERSION_MAJOR}.${TAPI_VERSION_MINOR}.${TAPI_VERSION_PATCH} -compatibility_version 1"
   )
